docs/framework/performance/lazy-initialization.md

```diff
@@ -78,35 +78,35 @@ manager: "wpickett"
   
- Exception caching is enabled when you use any <xref:System.Lazy%601?displayProperty=fullName> constructor that takes an initialization method (`valueFactory` parameter); for example, it is enabled when you use the `Lazy(T)(Func(T))`constructor. If the constructor also takes a <xref:System.Threading.LazyThreadSafetyMode> value (`mode` parameter), specify <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName> or <xref:System.Threading.LazyThreadSafetyMode?displayProperty=fullName>. Specifying an initialization method enables exception caching for these two modes. The initialization method can be very simple. For example, it might call the default constructor for `T`: `new Lazy<Contents>(() => new Contents(), mode)` in C#, or `New Lazy(Of Contents)(Function() New Contents())` in Visual Basic. If you use a <xref:System.Lazy%601?displayProperty=fullName> constructor that does not specify an initialization method, exceptions that are thrown by the default constructor for `T` are not cached. For more information, see the <xref:System.Threading.LazyThreadSafetyMode> enumeration.  
+ Exception caching is enabled when you use any <xref:System.Lazy%601?displayProperty=fullName> constructor that takes an initialization method (`valueFactory` parameter); for example, it is enabled when you use the `Lazy(T)(Func(T))`constructor. If the constructor also takes a <xref:System.Threading.LazyThreadSafetyMode> value (`mode` parameter), specify <xref:System.Threading.LazyThreadSafetyMode.ExecutionAndPublication?displayProperty=fullName> or <xref:System.Threading.LazyThreadSafetyMode.PublicationOnly?displayProperty=fullName>. Specifying an initialization method enables exception caching for these two modes. The initialization method can be very simple. For example, it might call the default constructor for `T`: `new Lazy<Contents>(() => new Contents(), mode)` in C#, or `New Lazy(Of Contents)(Function() New Contents())` in Visual Basic. If you use a <xref:System.Lazy%601?displayProperty=fullName> constructor that does not specify an initialization method, exceptions that are thrown by the default constructor for `T` are not cached. For more information, see the <xref:System.Threading.LazyThreadSafetyMode> enumeration.  
```

Comment:
instead of `<xref:System.Threading.LazyThreadSafetyMode.PublicationOnly?displayProperty=fullName>`, this should be <xref:System.Threading.LazyThreadSafetyMode.None?displayProperty=fullName>.  Exception caching is disabled with a **mode** of LazyThreadSafetyMode.PublicationOnly.